SmartDeviceLink/private/SDLProtocol.m

```diff
@@ -513,7 +513,7 @@ - (void)protocol:(SDLProtocol *)protocol didReceiveStartServiceACK:(SDLProtocolM
         switch (startServiceACK.header.serviceType) {
             case SDLServiceTypeRPC: {
                 SDLControlFramePayloadRPCStartServiceAck *startServiceACKPayload = [[SDLControlFramePayloadRPCStartServiceAck alloc] initWithData:startServiceACK.payload];
-
+                
```

Comment:
```suggestion

```

In Xcode, please go to preferences -> Text Editing -> Editing and turn on "Automatically trim trailing whitespace" and "Including whitespace-only lines". Thanks! If you just removed whitespace last time and didn't remove the entire line and I didn't notice, sorry!